creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -198,16 +224,34 @@ class SnapbackSM {
      * Initialize all queue processors
      */
 
-    // Initialize stateMachineQueue job processor
+    // Initialize stateMachineQueue job processor (aka consumer)
     this.stateMachineQueue.process(1 /** concurrency */, async (job) => {
-      const jobId = job.id
-      await redis.set('stateMachineQueueLatestJobStart', Date.now())
+      this.log('StateMachineQueue: Consuming new job...')
+      const { id: jobId } = job || { id: 'unknownId' }
```

Comment:
yeah it was able to reach that and never had `unknownId` -- it just got stuck on the first step (getNodeUsers) after the "BEGIN" phase was logged

Reply:
gotcha ok cool. at first thought it would be good to remove the unkonwnId part but maybe its useful to keep for a while and track if we ever see that in log